config/ipma/areas.php

```diff
@@ -0,0 +1,106 @@
+<?php
```

Comment:
Get rid of this and use the `Counties` we already have, instead.

Reply:
Districts, can't map to Counties.
The zone must be present , otherwise we will lose the ability to know if the warning is for the Continent / Madeira or Açores. Maybe zone its not the best name, any suggestion ?

Reply:
The array (map) is necessary, I can't remove otherwise i can't do the mapping areaId -> District.

Reply:
Use this, for example:
```php
return [
    'AVR' => 5,
    'BJA' => 24,
    'BRG' => 36,
    'BGC' => 49,
    'CBO' => 61,
    'CBR' => 73,
    'EVR' => 92,
    'FAR' => 106,
    'GDA' => 124,
    'LRA' => 140,
    'LSB' => 153,
    'PTG' => 177,
    'PTO' => 190,
    'STM' => 212,
    'STB' => 229,
    'VCT' => 239,
    'VRL' => 254,
    'VIS' => 277,

    'MCN' => 288, // Madeira - Costa Norte (São Vicente)
    'MCS' => 281, // Madeira - Costa Sul (Funchal)
    'MRM' => 287, // Madeira - R. Montanhosas (Santana)
    'MPS' => 289, // Madeira - Porto Santo

    'AOR' => 293, // Açores - Grupo Oriental (Ponta Delgada)
    'ACE' => 300, // Açores - Grupo Central (Calheta de São Jorge)
    'AOC' => 307, // Açores - Grupo Ocidental (Santa Cruz das Flores)
];
```

Tweak the Madeira/Açores map, since I went for the most central `County` on the map, and the IPMA sensors might not exactly be there. Probably ask the @METEO guys on Discord.

Reply:
Tx!